src/Dependency.jl

```diff
@@ -61,11 +61,11 @@ within which all of this will be installed.
 
 You may, alternately, provide a `Vector` of tuples containing the URLs and hashes
 """
-function Dependency{P <: Product}(name::AbstractString,
-                    results::Vector{P},
-                    cmds::Vector{Cmd},
-                    platform::Platform,
-                    prefix::Prefix = BinaryProvider.global_prefix)
+function Dependency(name::AbstractString,
+      results::Vector{P},
+      cmds::Vector{Cmd},
+      platform::Platform,
+      prefix::Prefix = BinaryProvider.global_prefix) where P <: Product
```

Comment:
@Keno why did it muck with the whitespacing here?

Reply:
It takes the length of `{P <: Product}` and unindents everything to match

Reply:
it heuristically considered the indent to be relative to the `(` and moved the lines with it. There's a change request to have it realign the block, but femtocleaner is out of commission right now